src/server/auth/utils.js

```diff
@@ -27,8 +44,34 @@ export const getAccessibleResourcesFromArboristasync = async (jwt) => {
     }
     throw new CodedError(data.error.code, data.error.message);
   }
-  const resources = data.resources ? _.uniq(data.resources) : [];
-  return resources;
+
+  const read = resourcePathsWithServiceMethodCombination(data, ['guppy', '*'], ['read', '*']);
+  const readResources = read.resources ? _.uniq(read.resources) : [];
+  return [readResources, data];
+};
+
+export const checkIfUserCanRefreshServer = async (data) => {
+  if (config.internalLocalTest) {
+    data = {
```

Comment:
consider fix the failed es-lint check here

Reply:
fxied